test/controllers/returns/NonExportedHumanMedicinesPlasticPackagingControllerSpec.scala

```diff
@@ -105,26 +108,9 @@ class NonExportedHumanMedicinesPlasticPackagingControllerSpec extends PlaySpec w
       captor.getValue.value mustBe Some(true)
     }
 
-    "display total plastic weight if DirectlyExportedComponentsPage and PlasticExportedByAnotherBusinessPage answer was no" in {
-
-      val result = createSut(userAnswer = Some(nonExportedAnswer.set(DirectlyExportedComponentsPage, false).success.value
-        .set(PlasticExportedByAnotherBusinessPage, false).success.value
-      ))
-        .onPageLoad(NormalMode)(FakeRequest(GET, nonExportedHumanMedicinesPlasticPackagingRoute))
-
-      status(result) mustBe OK
-
-      verify(mockView).apply(
-        ArgumentMatchers.eq(manufacturedAmount + importedAmount),
-        any(),
-        ArgumentMatchers.eq(NormalMode),
-        ArgumentMatchers.eq(false),
-        ArgumentMatchers.eq(false)
-      )(any(),any())
-
-    }
-
     "redirect GET to home page when DirectlyExportedComponentsPage and PlasticExportedByAnotherBusinessPage amount not found" in {
+      reset(nonExportedAmountHelper)
```

Comment:
Need this reset here? (It's already in beforeEach())

Reply:
(There a few more places too)

Reply:
Thank youu just removed the resets that weren’t needed